lib/utils/suppressBibs.js

```diff
@@ -6,16 +6,15 @@ const suppressBib = async (bib) => {
   const bibUri = await uriForRecordIdentifier(bib.nyplSource, bib.id, 'bib')
 
   const elasticClient = await client()
-  // Support a killswitch if something goes awry:
-  if (!process.env.CIRC_DELETE === 'true') return Promise.resolve()
 
-  logger.info(`deleting: ${bibUri}`)
+  logger.debug(`Deleting: ${bibUri}`)
 
-  return elasticClient.delete({
+  const opts = {
     index: process.env.ELASTIC_RESOURCES_INDEX_NAME,
     type: 'resource',
     id: bibUri
-  })
+  }
+  return elasticClient.delete(opts)
```

Comment:
is this not an async call? do we want to await it?

Reply:
It is an async call, but we don't need to do anything with the resolved value in this function, so it's fine to return the unresolved Promise